protocols/imap/src/main/java/org/apache/james/imap/processor/AbstractMailboxProcessor.java

```diff
@@ -108,7 +108,12 @@ final void doProcess(M message, ImapCommand command, String tag, Responder respo
             } else {
                 getMailboxManager().startProcessingRequest(ImapSessionUtils.getMailboxSession(session));
 
-                doProcess(message, session, tag, command, responder);
+                try {
+                    doProcess(message, session, tag, command, responder);
```

Comment:
You should not add a try block but add a catch clause in the existing one.
Because even if there is not a lot of non empty implementation `getMailboxManager().endProcessingRequest` can throw for example : `JPATransactionalMapper`

Reply:
Ok, already fixed